src/main/java/org/broadinstitute/hellbender/engine/LazyReadShard.java

```diff
@@ -195,14 +175,14 @@ public boolean containsStartPosition( final Locatable loc ) {
      * @param shardPadding desired shard padding; each shard's interval will be padded on both sides by this number of bases (may be 0)
      * @param readsSource data source for reads
      * @param dictionary sequence dictionary for reads
-     * @return List of {@link ReadShard} objects spanning the interval
+     * @return List of {@link LazyReadShard} objects spanning the interval
      */
-    public static List<ReadShard> divideIntervalIntoShards( final SimpleInterval interval, final int shardSize, final int shardPadding, final ReadsDataSource readsSource, final SAMSequenceDictionary dictionary ) {
+    public static List<LazyReadShard> divideIntervalIntoShards(final SimpleInterval interval, final int shardSize, final int shardPadding, final ReadsDataSource readsSource, final SAMSequenceDictionary dictionary ) {
```

Comment:
Your IDE seems to favor a strange style of spacing for parameter lists: `(like this )`